apps/interpreter/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@jvalue/interpreter",
+  "name": "@jvalue/jayvee",
```

Comment:
Does it make sense to name the interpreter **the** jayvee package?

Reply:
I don't think so, should be jayvee-interpreter at least.